src/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -362,7 +362,12 @@ public unsafe Vector2i Location
         public unsafe Vector2i Size
         {
             get => _size;
-            set => GLFW.SetWindowSize(WindowPtr, value.X, value.Y);
+            set
+            {
+                _size.X = value.X;
+                _size.Y = value.Y;
```

Comment:
You can just do `_size = value` you don't need to do separate assignments for x and y.

Reply:
ok, thx, I was warried about to change variable reference